server/test/services/ti/TrustedIntermediaryServiceTest.java

```diff
@@ -0,0 +1,242 @@
+package services.ti;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static play.api.test.CSRFTokenHelper.addCSRFToken;
+import static play.test.Helpers.fakeRequest;
+
+import auth.ProfileFactory;
+import com.google.common.collect.ImmutableMap;
+import controllers.WithMockedProfiles;
+import forms.AddApplicantToTrustedIntermediaryGroupForm;
+import models.Account;
+import models.Applicant;
+import models.TrustedIntermediaryGroup;
+import org.junit.Before;
+import org.junit.Test;
+import play.data.Form;
+import play.data.FormFactory;
+import play.mvc.Http;
+import repository.UserRepository;
+
+public class TrustedIntermediaryServiceTest extends WithMockedProfiles {
+
+  private UserRepository repo;
+
+  private TrustedIntermediaryService service;
+  private FormFactory formFactory;
+  private ProfileFactory profileFactory;
+
+  @Before
+  public void setup() {
+    repo = instanceOf(UserRepository.class);
+    service = instanceOf(TrustedIntermediaryService.class);
+    formFactory = instanceOf(FormFactory.class);
+    profileFactory = instanceOf(ProfileFactory.class);
+    Applicant managedApplicant = createApplicant();
+    createTIWithMockedProfile(managedApplicant);
+    repo = instanceOf(UserRepository.class);
+    profileFactory.createFakeTrustedIntermediary();
+  }
+
+  @Test
+  public void testAddClientWithInvalidDOB() {
```

Comment:
suggestion (blocking): please use the test method naming convention `methodUnderTest_contextForTheTest` e.g. `addClient_withInvalidDob`